teams/templates/includes/member.html

```diff
@@ -0,0 +1,13 @@
+{% load static %}
+
+<div class="card card-no-border team-member-card ml-auto mr-auto">
+
+    <a href="{% url 'show_profile' user_id=member.id %}"><h5 class="card-text text-center mb-3">{{member.slack_display_name}}</h5></a>
+    {% if member.profile_image %}
+        <img class="card-img-top team-member-img"
+            src="{{member.profile_image}}" alt="Profile Image">
+    {% else %}
+    <img class="card-img-top team-member-img"
+        src="{% static 'img/profiles/profile_cat.png' %}" alt="Profile Image">
+    {% endif %}
+</div>
```

Comment:
Missing blank line at end of the file here.

Reply:
Added blank line.